stories/BaseDataSearch.vue

```diff
@@ -0,0 +1,53 @@
+<template>
+  <ReactiveBase app="good-books-ds" credentials="nY6NNTZZ6:27b76b9f-18ea-456c-bc5e-3a5263ebc63d" >
+  <DataSearch
+    className="result-list-container"
+    categoryField="authors.raw"
+    componentId="BookSensor"
+    :dataField="['original_title', 'original_title.search']"
+    v-bind='subProps'
+    v-on='subEvents'
+  />
+  <ReactiveList
```

Comment:
@jyash97 Please use the same `BaseReactiveForm` component to avoid the code duplication.